Formula/libfido2.rb

```diff
@@ -0,0 +1,46 @@
+class Libfido2 < Formula
+  desc "Provides library functionality for FIDO U2F & FIDO 2.0, including USB"
+  homepage "https://developers.yubico.com/libfido2/"
+  url "https://github.com/Yubico/libfido2/archive/1.3.0.tar.gz"
+  sha256 "0b2e3671c4c5d42fd5604a08e45f89f49592b97cf66d7d3bfbc7e6a4d5a0fec7"
+  depends_on "cmake" => :build
+  depends_on "mandoc" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libcbor"
+  depends_on "openssl@1.1"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "man_symlink_html"
+      system "make", "man_symlink"
+      system "make", "install"
```

Comment:
@xanderlent Just to make sure I followed the discussion: If you build a release build there is no `make test` target? Or `ctest`?

It's possible you [may need to add `-DBUILD_TESTING:BOOL=ON`](https://cmake.org/cmake/help/v3.16/module/CTest.html) to the CMake arguments to setup the tests for a `Release` build. That is the canonical way (or a canonical way) for CMake projects to enable and disable tests.

Reply:
I'll give it a try. I got my current understanding from an upstream bug which described the situation and some cursory reading of CMake files.

Unfortunately I don't know anything about CMake, so understanding upstream CMake files has been a challenge.

Reply:
@xanderlent: I spend more of my time than I would care to admit dealing with cmake. I just looked through the upstream build system. It appears that the only testing they do is fuzzing on linux and regression testing during the build. It looks like this regression testing is enabled by default. To be extra sure you could add a `make regress` just to verify the tests were run. Other than those, and the fuzzing they do on linux, there are no other tests present in the default branch.

Reply:
__TLDR:__ ignore this suggestion. My other suggestion about `make regress` is probably redundant, but you can accept it if we want to be sure that those regression tests were actually run during the build, which it appears they should be by default.